src/ResourceManager/DevTestLab/Commands.DevTestLab/AzureRM.DevTestLab.psd1

```diff
@@ -0,0 +1,88 @@
+#  
```

Comment:
This is the old format psd1 file, you'll need to use the new format psd1 file, for example AzureRm.Compute\AzureRm.Compute.psd1